physcraper/__init__.py

```diff
@@ -1812,6 +1809,44 @@ def run_blast_wrapper(self):  # TODO Should this be happening elsewhere?
                                                                                                             last_blast))
         self._blasted = 1
 
+    def get_acc_from_blast(self, query_string):
```

Comment:
Will these only work with a local blast database? If so would be good to make that clear in function name - (e.g. get_acc_from_local_blast). 
Is there any way to test within this function that the value is actually an accession number? (e.g. in case there is an error in the query string that is fed in). Or to test that the query string has some specific characteristics?

Reply:
No, looks like it also works for web blasting. Adjusted doc string accordingly.
Yeah, not sure how else to test, that the right string was fed in.

Reply:
Sounds good.